triangle-function.js

```diff
@@ -0,0 +1,19 @@
+// Napisz funkcję triangle(), która w zależności od liczby argumentów obliczy
+// pole trójkąta na podstawie długości podstawy i wysokości lub na podstawie długości wszystkich jego boków.
+
+
+function triangle(a, h, c) {
```

Comment:
h is not a good name for this parameter since it doesn't have to be a height. In this situation a, b and c would be the best in my opinion.